delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/encoder/JsonLdEncoder.scala

```diff
@@ -61,6 +61,19 @@ trait JsonLdEncoder[A] {
       ntriples <- IO.fromEither(graph.toNTriples)
     } yield ntriples
 
+  /**
+    * Converts a value of type ''A'' to [[NQuads]] format.
+    *
+    * @param value the value to be converted to n-quads format
+    */
+  def nquads(
```

Comment:
missing tests for this

Reply:
No, it is not missing. The tests are present in DeltaDirectivesSpec and also in GraphSpec (this method just makes a call to graph.toNQuads)

Reply:
There are specific tests for this in `ch.epfl.bluebrain.nexus.delta.rdf.jsonld.JsonLdEncoderSpec`

Reply:
I've added tht test. even thought the actual useful tests are in GraphSpec, as expected